src/main/java/dev/fiki/forgehax/main/Common.java

```diff
@@ -85,33 +88,42 @@ static EventBus getEventBus() {
   }
 
   //
-  // forge
+  // Forge
   //
 
   static ClassLoader getLauncherClassLoader() {
     return FMLLoader.getLaunchClassLoader();
   }
 
   //
-  // minecraft
+  // MC
   //
 
   static GameSettings getGameSettings() {
     return MC.gameSettings;
   }
 
-  static Stream<Entity> worldEntities() {
+  //
+  // World entities
+  // // TODO: this stuff might need to be try catched since mc doesn't have it synchronized
+  //
+
+  static Stream<AbstractClientPlayerEntity> getWorldPlayers() {
+    return !isInWorld() ? Stream.empty() : getWorld().getPlayers().stream();
+  }
+
+  static Stream<Entity> getWorldEntities() {
     return !isInWorld() ? Stream.empty()
         : StreamSupport.stream(getWorld().getAllEntities().spliterator(), false);
   }
 
-  static Stream<TileEntity> worldTileEntities() {
+  static Stream<TileEntity> getWorldTileEntities() {
```

Comment:
I know this is proper naming conventions but why refactor in this PR?

Reply:
it's my bad habit to always include stuff that don't belong to the main PR topic lol, if you don't want I can do another one for this specifically

Reply:
also for this getter specifically and related I'd like to not return a stream but just the list itself as you could use it for more stuff then

Reply:
Yeah that's fine. Just create another function though